lib/licensed/sources/nuget.rb

```diff
@@ -175,7 +175,8 @@ def enabled?
       # easily machine readable and doesn't contain everything we need.
       def enumerate_dependencies
         json = JSON.parse(project_assets_file)
-        nuget_packages_dir = json["project"]["restore"]["packagesPath"]
+        nuget_packages_dirs = [json["project"]["restore"]["packagesPath"]]
+        nuget_packages_dirs << json["project"]["restore"]["fallbackFolders"]
```

Comment:
Rather than using `json["project"]["restore"]["packagesPath"]` and `json["project"]["restore"]["fallbackFolders"]`, might it be appropriate to use `json["packageFolders"]` which appears to already combine them?

I don't know why both methods exist for determining the set of paths or how supported they are; might want to ask NuGet.

Reply:
discussion here for historical reference on packagesPath + fallbackFolders being the correct approach: https://github.com/dotnet/sdk/issues/11773